public/app/frontend/src/documentation/development/Setup.mdx

```diff
@@ -1,11 +1,27 @@
 import { Meta } from '@storybook/blocks';
+import { getReviewBlock } from '../../../.storybook/utils/documentation/review-block';
+
+{/* Update the lastReview value in YYYY-MM-DD format when this page is reviewed */}
+export const lastReview = '2024-12-06';
 
 <Meta title="Development/Setup" />
 
-TODO: write later as structure/docker integration might change
+# Setup
+
+## Frontend only
+Our component library is designed so that it can be worked on without running the backend. If you're working on a
+component and don't need to integrate it with WordPress, then you should only need to:
+
+1. Checkout the justice-gov-uk repository here: https://github.com/ministryofjustice/justice-gov-uk.
+2. `cd` into the `public/app/frontend` directory.
+3. Run `npm install` to install the required packages.
+4. Run `npm run storybook` to start Storybook. It should open automatically in your browser at `localhost:6006`.
+
```

Comment:
It's personal view that I try not to run node without using docker. There's a chance that a malicious dependency can run arbitrary scripts during install or runtime.

For that reason I don't have node or npm installed. Could you add this alternative command (or similar) to the docs?

`docker compose run --service-ports node node-frontend`

It will start the node container only, and then run the `node-frontend` task from the makefile. Storybook will then be available at `localhost:6006` or `storybook.justice.docker`.

Reply:
Thanks @EarthlingDavey, that's good to know. I've updated the docs to include that line.